K번째수.js

```diff
@@ -0,0 +1,21 @@
+function solution(array, commands) {
+    let i = 0;
+    let arr = 0;
+    let rarr = 0;
+    let answer = [];
```

Comment:
```suggestion
    const answer = [];
```
배열의 경우 재선언되지 않는 이상 변하지 않기 때문에 const로 선언해주세요